src/VirtoCommerce.DemoSolutionFeaturesModule.Web/Scripts/blades/product-part-list.tpl.html

```diff
@@ -0,0 +1,89 @@
+<div class="blade-static">
+    <div class="form-group">
+        <div class="form-input __search">
+            <input ng-model="filter.keyword"
+                   placeholder="{{'platform.placeholders.search-keyword' | translate}}"
+                   ng-keyup="$event.which === 13 && filter.criteriaChanged()" />
```

Comment:
Add debounce if possible

Reply:
Doesn't need here, because `13` is a code for `Enter` key. I added constant for that in js to make it clear.